content/200-concepts/100-components/01-prisma-schema/07-referential-actions.mdx

```diff
@@ -0,0 +1,329 @@
+---
+title: 'Referential actions'
+metaTitle: 'Referential actions'
+metaDescription: 'Referential actions let you define the update and delete behavior of related models on the database level'
+tocDepth: 2
+preview: true
+---
+
+<TopBlock>
+
+As of [2.26.0](https://github.com/prisma/prisma/releases/tag/2.26.0] Prisma now supports _referential actions_ as a Preview feature part of the Prisma Schema Language.
+
```

Comment:
I don't think we need 'part of the Prisma Schema Language' here (we rarely mention PSL as a concept), and I think we should include a one-liner about what referential actions are at the very top of the page and an example as soon as possible. Maybe something like what you have a bit further down - 

"In 2.26.0 and later, you can define referential actions on your relation fields. Referential actions determine what should happen to a record when a **related record** is deleted or updated. In the following example:

* Deleting the referenced `author` deletes the `Post` (`onDelete: Cascade`)
* Updating the referenced `author` updates the `authorId` foreign key on the `Post` (`onUpdate: Cascade`)

```
model Post {
    id        Int     @id @default(autoincrement())
    title     String
    authorId  Int
    author    User    @relation(fields: [authorId], references: [id], onDelete: Cascade, onUpdate: Cascade)
 }
```